docs/source/AdministratorGuide/ServerInstallations/environment_variable_configuration.rst

```diff
@@ -40,3 +40,9 @@ DIRACSYSCONFIG
 
 DISABLE_WATCHDOG_CPU_WALLCLOCK_CHECK
   This variable only makes sense for DIRAC pilots. When set, the wallclock check done within the watchdog is disabled.
+
+DIRAC_X509_HOST_CERT
+  Defines the location of the host certificate, and takes precedence over CS options. You should not set this in your DIRAC ``bashrc``, prefer the CS options.
```

Comment:
Do you mean

```suggestion
  Defines the location of the host certificate, and takes precedence over CS options. Should you not set this in your DIRAC ``bashrc``, the CS option would be preferred.
```
?

Reply:
No no, I really mean that it should not be set in the bashrc and we should prefer the CS options. These env variables are really to overwrite what is in the CS

Reply:
OK, can you re-phase/add more info? Sorry I find it not-so-clear.

Reply:
better ?